crates/store/re_video/src/decode/ffmpeg.rs

```diff
@@ -151,6 +154,61 @@ impl FfmpegProcessAndListener {
             });
         }
 
+        // By default play it safe: let ffmpeg convert to rgba.
+        let mut pixel_format = PixelFormat::Rgba8Unorm;
+        let mut ffmpeg_pix_fmt = "rgba";
+
+        // There might be extensions to the SPS (`NalUnitType::SequenceParameterSetExt`), ignore those.
+        let mut sps_units =
+            avcc.avcc.sequence_parameter_sets.iter().filter(|sps| {
+                NalHeader(sps.bytes[0]).unit_type() == NalUnitType::SequenceParameterSet
+            });
+        if let Some(sps_unit) = sps_units.next() {
+            if sps_units.next().is_some() {
+                // This is rather strange. Must mean that some pictures refer to one SPS and some to another!
+                // We don't know what to do with this.
+                re_log::trace_once!("Found more than one sequence parameter set (SPS) in the AVCC box of {debug_name}.");
```

Comment:
I suggest we use `warn` in debug builds. We could add some helper for that in `re_log`

Reply:
after refactoring the whole thing to a method and - sparked by that - modularizing stuff more, I decided to just always warn. I'd rather have people complain about this than not finding these issues in our understanding!